roles/rsyslog/tasks/main.yaml

```diff
@@ -208,13 +215,13 @@
         state: stopped
       when:
         - not rsyslog_enabled | bool
-        - not use_rsyslog_image | default(false) | bool
+        - not rsyslog_in_image | default(false) | bool
 
-  when: rsyslog_failed_validation | d(false) == false
+  when: not __rsyslog_failed_validation | d(false) | bool
```

Comment:
do you need the bool filter here? This filter converts strings to booleans. Is the value supposed to be a string?

Reply:
Fixed.